Sprint-1/exercises/decimal.js

```diff
@@ -1,9 +1,21 @@
-const num = 56.5678;
-
-// You should look up Math functions for this exercise https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math
-
-// Create a variable called wholeNumberPart and assign to it an expression that evaluates to 56 ( the whole number part of num )
-// Create a variable called decimalPart and assign to it an expression that evaluates to 0.5678 ( the decimal part of num )
-// Create a variable called roundedNum and assign to it an expression that evaluates to 57 ( num rounded to the nearest whole number )
-
-// Log your variables to the console to check your answers
+const num = 56.5678;

```

Comment:
This is a good attempt 😄 But I think we can review a few concepts here.

Let's pay special attention to the requirements of the exercise: they say "Create a variable called decimalPart", but are we satisfying this requirement here? 🤔 Take a look at other examples of "creating a variable" in the prep work: https://programming.codeyourfuture.io/structuring-data/sprints/1/prep/#variables